brevo/assets/monitors/brevo_high_block_rate_by_subject.json

```
@@ -0,0 +1,92 @@
+{
+	"version": 2,
+	"created_at": "2024-09-11",
+	"last_updated_at": "2024-09-11",
+	"title": "[Brevo] High Block Rate by Subject",
+	"description": "Monitors high block rates by subject.",
```

The description should follow [best practice guidelines](https://docs.datadoghq.com/developers/integrations/create-an-integration-recommended-monitor/#description):
- This is not a copy of the title.
- Define the problem stated by the title.
- Answer why this is an issue worth alerting on.
- Describe the impact of the problem.

```suggestion
	"title": "Block rate by subject is higher than usual",
	"description": "Monitors high block rates by subject.",
```